src/spyglass/common/common_usage.py

```diff
@@ -408,3 +414,70 @@ def write_export(
         )
 
         # TODO: export conda env
+
+    def compile_dandiset(
+        self, key: dict, dandiset_id: str, dandi_api_key: str = None
+    ):
+        """Compile a Dandiset from the export.
+        Parameters
+        ----------
+        key : dict
+            ExportSelection key
+        dandiset_id : str
+            Dandiset ID generated by the user on the dadndi server
+        dandi_api_key : str, optional
+            API key for the dandi server. Optional if the environment variable
+            DANDI_API_KEY is set.
+        """
+        key = (self & key).fetch1("KEY")
+        paper_id = (self & key).fetch1("paper_id")
+        from .common_dandi import DandiPath, validate_dandiset
+
+        # make a temp dir with symbolic links to the export files
+        source_files = (self.File() & key).fetch("file_path")
+        destination_dir = f"{export_dir}/dandiset_{paper_id}"
```

Comment:
My gut says to organize these as 

```
export_dir
|- paper_id1
     |- export.sh
     |- export.sql
     |- dandi_folder
|- paper_id2
```
so that there's only one top level item per paper